javascript/ql/src/Statements/UselessComparisonTest.ql

```diff
@@ -0,0 +1,60 @@
+/**
+ * @name Useless comparison test
+ * @description A comparison that always evaluates to true or always evaluates to false may
+ *              indicate faulty logic and dead code.
+ * @kind problem
+ * @problem.severity warning
+ * @id js/useless-range-check
```

Comment:
Is the slight discrepancy between query name and ID (comparison test vs range check) intentional?

Reply:
That was a leftover from renaming the query, fixed it.